crates/subspace-archiving/src/lib.rs

```diff
@@ -17,6 +17,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![feature(drain_filter)]
 
+extern crate core;
```

Comment:
I don't think `extern crate core` is actually necessary, `core` always works, it is only `alloc` that you need in `no_std` environment.